WARNING: leap-second auto-update failed due to the following exception: RuntimeError('Cache is locked after 5.00 s. This may indicate an astropy bug or that kill -9 was used. If you want to unlock the cache remove the directory /home/data/bl407/jwiniarczyk/.astropy/cache/download/py3/lock. Lock claims to be held by process 6608.',) [astropy.time.core]
The chain is shorter than 50 times the integrated autocorrelation time for 10 parameter(s). Use this estimate with caution and run a longer chain!
N/50 = 6;
tau: [33.83644082 31.18573112 31.23682515 32.28237858 33.04158239 30.26957321
 28.68181752 37.6273348  38.32136879 32.57614202]
WARNING:root:Too few points to create valid contours
